kolibri/auth/serializers.py

```diff
@@ -45,6 +45,13 @@ class Meta:
         fields = ('id', 'username', 'full_name', 'password', 'facility', 'roles')
 
 
+class FacilityUsernameSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = FacilityUser
+        fields = ('id', 'username', )
```

Comment:
On the Model? Or here somewhere? (don't think we'd want to mess with the model itself)